packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -679,11 +679,13 @@ DashboardLayout.propTypes /* remove-proptypes */ = {
         iconButton: PropTypes.object,
         signInButton: PropTypes.object,
         signOutButton: PropTypes.object,
+        userDetailsContainer: PropTypes.object,
       }),
       slots: PropTypes.shape({
-        menuItems: PropTypes.elementType,
+        content: PropTypes.elementType,
```

Comment:
Maybe the name `content` is too generic, but it's just my opinion, I don't mind it.